saliency/integrated_gradients.py

```diff
@@ -14,20 +14,40 @@
 
 """Utilities to compute an IntegratedGradients SaliencyMask."""
 
+from .base import CallModelSaliency
+from .base import OUTPUT_GRADIENTS
 import numpy as np
-from .base import GradientSaliency
 
-class IntegratedGradients(GradientSaliency):
-  """A SaliencyMask class that implements the integrated gradients method.
+
+class IntegratedGradients(CallModelSaliency):
+  """A CallModelSaliency class that implements the integrated gradients method.
 
   https://arxiv.org/abs/1703.01365
   """
 
-  def GetMask(self, x_value, feed_dict={}, x_baseline=None, x_steps=25):
+  def GetMask(self, x_value, call_model_function, call_model_args=None,
+              x_baseline=None, x_steps=25, batch_size=1):
```

Comment:
Does the batch_size '1' mean that the first dimension of the batched input is one or is the dimension absent?

Reply:
The first dimension is 1. This makes the input passed to call_model_function more uniform, meaning it should always be 4 dimensions (if the images are RGB, the batch would be 3 dimensions if grayscale). Maybe we could verify the input to make sure the batch size is always int>0?

Reply:
Added checks here with format_call_model_data